### ISSUE TITLE
Tag count of deleted videos does not update / tag is not removed from tag tray when all tags are removed

### ISSUE THREAD

**Issue by Xenonfish** [whyboris/Video-Hub-App#860]
I added a manual tag to a batch of 79 videos in a folder, and then reviewed each video and decided to delete 62 of those videos. I enabled the dangerously delete option in settings to do this by right click > delete.

When I opened the tag tray the manual tag was still showing quantity of 79 (it should of been 17). I thought I'd manually remove all of the remaining 17 tags from the videos left over thinking that this may remove the tag from the tag tray, but it didn't - the tag still shows 62 and when I click on the tag, the results window shows "No Results Found" 

I've tried a re-scan of the parent folder under settings > current hub but this didn't work. 

Am I overlooking something / is there a workaround ?

Thanks

**Comment by whyboris** [comment]
Thank you for the bug report! The tag count _should_ update after closing out the application and re-opening it (I am guessing the "change detection" isn't getting triggered upon deleting a file). 

You've definitely found a bug - I just hope it auto-fixes itself when closing/opening the app/hub again 🤞 

_Did the erroneous count fix itself after closing and re-opening the app (or hub)?_

**Comment by Xenonfish** [comment]
You're correct, I just needed to close and re-open the app which solved it :) No real issue to restart but is the change detection meant to do this automatically? Are there other functions that will also require a restart to update / sync the app?

Thanks
